vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,705 @@
+
+---
+title: "Introduction to GenomicFiles"
+author: Valerie Obenchain,Michael Love,Lori Shepherd,Martin Morgan
+date: March 04, 2023
+package: GenomicFiles
+vignette: >
+  %\VignetteIndexEntry{Introduction to GenomicFiles}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+# Introduction
+
+This vignette illustrates how to use the `GenomicFiles` package for distributed computing across files. The functions in `GenomicFiles` manipulate and combine data subsets via two user-supplied functions, `MAP` and `REDUCE`. These are similar in spirit to `Map` and `Reduce` in base R. Together they provide a flexible interface to extract, manipulate, and combine data. Both functions are executed in the distributed step, which means results are combined on a single worker, not across workers.
+
+We assume the reader has some previous experience with R and with basic manipulation of ranges objects such as `GRanges` and `GAlignments` and file classes such as `BamFile` and `BigWigFile`. See the vignettes and documentation in `GenomicRanges`, `GenomicAlignments`, `Rsamtools` and `rtracklayer` for an introduction to these classes.
```

Comment:
ohk !! Thanku

Reply:
I want to clarify the links a bit more because while you were able to generate the links, there's a better way to do it and  you won't see the effect until your vignette has `BiocStyle` included in the yaml yeader.

We use the functions mentioned in the BiocStyle documentation to generate the links. We do this because paths can later change, vignettes can always be knitted again, etc. If you look at Style Macros in BiocStyle, it says that you should use

     `r Biocpkg("GenomicFiles")` 

if you want knitr to generate a link in the HTML document. Please make this change for all Bioconductor packages in the vignette.